Gems/ScriptCanvas/Code/Include/ScriptCanvas/Data/BehaviorContextObject.cpp

```diff
@@ -124,6 +96,12 @@ namespace ScriptCanvas
             : BehaviorContextObjectPtr(aznew BehaviorContextObject(reference, GetAnyTypeInfoReference(typeID), referenceFlags));
     }
 
+    void BehaviorContextObject::Deserialize(BehaviorContextObject& target, const AZ::BehaviorClass& behaviorClass, AZStd::any& source)
+    {
+        target.m_object = AZStd::move(AZStd::any(AZStd::any_cast<void>(&source), GetAnyTypeInfoObject(behaviorClass)));
```

Comment:
Nitpick: the `AZStd::move` has no effect here because a new instance of `AZStd::any` is created which is by definition an r-value already.